nrf_security/Kconfig

```diff
@@ -115,6 +115,7 @@ config MBEDTLS_ENABLE_HEAP
 config MBEDTLS_HEAP_SIZE
 	int "Heap size for mbed TLS"
 	default 15360 if OPENTHREAD_NRF_SECURITY
+	default 65536 if BUILD_WITH_TFM
```

Comment:
Warning, when `MBEDTLS_HEAP_SIZE` has a prompt, then the default value for the heap will not change if the user uses menuconfig to enable `BUILD_WITH_TFM`.

Reply:
Have to think  about this one...

Reply:
I am afraid to make this heap promptless as it may very well be set by default (it is a very "settable configuration") We can discuss a different scheme than this after merging